src/LIBSVM.jl

```diff
@@ -368,12 +374,22 @@ K(t_i, x_2), ..., K(t_i, x_l)]`, where `t_i` is `i`-th testing instance and
 `x_j` is `j`-th training instance. For linear kernel, `M' * T` produces the
 correct matrix, where columns of `M` contain the training instances and columns
 of `T` the testing instances.
+Alternatively a shape of `(k, n)` is also accepted, where `k` is the number of
+support vectors of the model. The order should be the same as in `model.SVs.indices`.
```

Comment:
I like the idea of the potential performance increase but I figured that it would most likely be overly complicated. I'll remove this part of the change. Thank you for the comment.

Reply:
Additionally, if performance is indeed critical, users could still construct a gram matrix this way themselves.